Kitodo/src/main/java/org/kitodo/services/image/ImageService.java

```diff
@@ -100,4 +103,20 @@ Image getScaledWebImage(URI imageFileUri, double percent) throws IOException {
     Image getSizedWebImage(URI imageFileUri, int pixelWidth) throws IOException {
         return imageManagement.getSizedWebImage(imageFileUri, pixelWidth);
     }
+
+    /**
+     * Return singleton variable of type FilterService.
```

Comment:
You are returning an instance of  `ImageService` class, so you should adjust the JavaDoc.

Reply:
Changed

Reply:
Changed